src/parser/json/json_parser.py

```diff
@@ -1,83 +1,22 @@
-# Copyright (c) 2022 spdx contributors
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#     http://www.apache.org/licenses/LICENSE-2.0
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
+#  Copyright (c) 2022 spdx contributors
```

Comment:
?

Reply:
That seems to be a PyCharm issue. I have the copyright text there without any leading whitespaces, but it includes two (instead of one) spaces in each line. Shouldn't pose a problem, though, I think, as changing a file in a manner that the copyright text gets edited is rare.